### PR TITLE
[REVIEW] Update RAPIDS Demo webpage

### PR DESCRIPTION
**Pull request by taureandyernv**
added new data sources, new content sources, and updated prereqs to 0.19

### CODE REVIEW

**Comment by taureandyernv** [comment]
Thanks @Salonijain27 for checking the directory structures or me!

@ajschmidt8 can we please merge these changes?

File: demos/rapids-demo.md [hunk 1]
```diff
@@ -69,17 +69,17 @@ The tag naming scheme for RAPIDS images incorporates key platform details into t
 
 To get the latest RAPIDS version of a specific platform combination, simply exclude the RAPIDS version.  For example, to pull the latest version of RAPIDS for the `runtime` image with support for CUDA {{ CUDA_VER }}, Python {{ PYTHON_VER }}, and Ubuntu 18.04, use the following tag:
 ```
-cuda{{ CUDA_VER }}-runtime-{{ UBUNTU_VER }}-py{{ PYTHON_VER }}
+{{ STABLE_VER }}-cuda{{ CUDA_VER }}-runtime-{{ UBUNTU_VER }}-py{{ PYTHON_VER }}
```
Comment (ajschmidt8) [inline demos/rapids-demo.md:72]:
```suggestion
cuda{{ CUDA_VER }}-runtime-{{ UBUNTU_VER }}-py{{ PYTHON_VER }}
```
This change contradicts the line above it, which states that the latest RAPIDS images can be obtained by omitting the RAPIDS version from the image name. Please revert.

File: demos/rapids-demo.md [hunk 2]
```diff
@@ -69,17 +69,17 @@ The tag naming scheme for RAPIDS images incorporates key platform details into t
 
 To get the latest RAPIDS version of a specific platform combination, simply exclude the RAPIDS version.  For example, to pull the latest version of RAPIDS for the `runtime` image with support for CUDA {{ CUDA_VER }}, Python {{ PYTHON_VER }}, and Ubuntu 18.04, use the following tag:
```
Comment (ajschmidt8) [inline demos/rapids-demo.md:70]:
```suggestion
To get the latest RAPIDS version of a specific platform combination, simply exclude the RAPIDS version.  For example, to pull the latest version of RAPIDS for the `runtime` image with support for CUDA {{ CUDA_VER }}, Python {{ PYTHON_VER }}, and Ubuntu {{ UBUNTU_VER }}, use the following tag:
```

Comment (ajschmidt8) [reply demos/rapids-demo.md:70]: Can you add this change to your PR?

File: demos/rapids-demo.md [hunk 3]
```diff
@@ -69,17 +69,17 @@ The tag naming scheme for RAPIDS images incorporates key platform details into t
...
 ```
-cuda{{ CUDA_VER }}-runtime-{{ UBUNTU_VER }}-py{{ PYTHON_VER }}
+{{ STABLE_VER }}-cuda{{ CUDA_VER }}-runtime-{{ UBUNTU_VER }}-py{{ PYTHON_VER }}
 ```
 
-Many users do not need a specific platform combination but would like to ensure they're getting the latest version of RAPIDS, so as an additional convenience, a tag named simply `latest` is also provided which is equivalent to `cuda{{ CUDA_VER }}-runtime-ubuntu16.04-py{{ PYTHON_VER }}`.
+Many users do not need a specific platform combination but would like to ensure they're getting the latest version of RAPIDS, so as an additional convenience, a tag named simply `latest` is also provided which is equivalent to `{{ STABLE_VER }}-cuda{{ CUDA_VER }}-runtime-ubuntu18.04-py{{ PYTHON_VER }}`.
```
Comment (ajschmidt8) [inline demos/rapids-demo.md:75]:
```suggestion
Many users do not need a specific platform combination but would like to ensure they're getting the latest version of RAPIDS, so as an additional convenience, a tag named simply `latest` is also provided which is equivalent to `{{ STABLE_VER }}-cuda{{ CUDA_VER }}-runtime-ubuntu{{ UBUNTU_VER }}-py{{ PYTHON_VER }}`.
```
we should probably make the Ubuntu version a variable like the others here.

**Review by ajschmidt8 — CHANGES_REQUESTED** [review]

File: demos/rapids-demo.md (hunk 3 above)
Comment (taureandyernv) [reply demos/rapids-demo.md:75]: agreed!

**Review by ajschmidt8 — APPROVED** [review]

**Pull request merged**